src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -689,13 +694,175 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
             set selectedTreatmentOption(o: any) {
                 this._selectedTreatmentOption = o;
             },
-            _selectedGeneOption: undefined,
-            _selectedGenesetOption: undefined,
-            _selectedTreatmentOption: undefined,
-            _dataType: undefined,
-            _mutationCountBy: undefined,
-            _logScale: false,
-            _selectedDataSourceOption: undefined,
+
+            get _selectedGeneOption() {
+                const param = vertical
```

Comment:
where does vertical come from?

Reply:
this is within a private helper method that creates the axis selection object with the parameter `vertical:boolean`